ipa-multipoint/ipa_multipoint_jni/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "ipa_multipoint_jni"
+version = "0.1.0"
+description = "IPA Multipoint Crypto Library"
+license = "Apache-2.0"
+authors = ["Antoine Toulme <antoine@lunar-ocean.com>"]
+repository = "https://github.com/hyperledger/besu-native"
+edition = "2018"
+
+[dependencies]
+ipa-multipoint = { path = "../ipa_multipoint" }
+bandersnatch = "0.1.1"
+jni = { version = "0.19.0", features = ["invocation"] } # We use invocation in tests.
+rand_chacha = "0.3.1"
+rand_core = "0.6.3"
+rand = "0.8.5"
```

Comment:
It seems that these libraries are no longer needed - better remove them as downloading them will slow the build down.